app/Console/Commands/TestCommand.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class TestCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'command:name';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Command description';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        eval($this->option('test'));
+
+        for ($i = 0; $i < $length; $i--) { // Noncompliant
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: "$i" is decremented and will never reach "stop condition". ([php:S2251](https://sonarcloud.io/organizations/dbmeneses-github/rules#rule_key=php%3AS2251))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=monica-dm-test&pullRequest=1&issues=AWSsghyll6vpliCetuqN&open=AWSsghyll6vpliCetuqN)

[ISSUE_KEY]: AWSsghyll6vpliCetuqN